Library/Formula/carthage.rb

```diff
@@ -1,6 +1,6 @@
 class Carthage < Formula
   homepage "https://github.com/Carthage/Carthage"
-  url "https://github.com/Carthage/Carthage.git", :tag => "0.6.2",
+  url "https://github.com/Carthage/Carthage.git", :tag => "0.6.3",
```

Comment:
Can you add a `:revision => ` indicating the revision this tag points to? Thanks!